packages/jest-snapshot/src/utils.ts

```diff
@@ -76,14 +74,14 @@ const validateSnapshotVersion = (snapshotContents: string) => {
   return null;
 };
 
-function isObject(item) {
+function isObject(item: any): boolean {
```

Comment:
```suggestion
function isObject(item: unknown): boolean {
```

If that gives you type errors, keep `any`